### DISCUSSION TITLE
Hello, I uploaded my project to github, but some photos do not appear in the hosting of github, why?

### DISCUSSION THREAD

**Discussion by omervordex** [community/community#108513] [Packages] [answered]
### Select Topic Area

Question

### Body

Hello, I uploaded my project to github, but some photos do not appear in the hosting of github, why?
Please help me (should I get a paid membership to solve this problem?)

**Comment by omervordex** [comment]
![Omer-Wealthome - Google Chrome 2_16_2024 6_07_07 PM](https://github.com/community/community/assets/111154760/01585e9b-9522-427b-b7ec-83a2eac0b444)
![Omer-Wealthome - Google Chrome 2_16_2024 6_07_17 PM](https://github.com/community/community/assets/111154760/d3b5de58-a161-4ec8-ab6a-5c126bfa78b1)

**Reply by hyperdgx** [reply]
Example: ```<img src="image.png" alt="image">```
Solution: Add ./ or ./folder, ```<img src="./image.png" alt="image">``` or ```<img src="./folder/image.png" alt="image">```

**Comment by Jossyboydgenius** [comment]
You don't need to get membership to resolve that.

**Comment by omervordex** [comment]
> You don't need to get membership to resolve dat.

How can I fix. vs code appears on live server but not on github pages

**Comment by Cizr** [comment, marked as answer]
Uploading your project to GitHub should not require a paid membership for images to display. However, GitHub has some limitations on file size and might not support certain file types for display...also what you trying to say is when you host the website the images do not appear right? maybe check "Relative Paths" or it could be "Deployment" issue

**Reply by omervordex** [reply]
Thanks for information.
